.travis.yml

```diff
@@ -38,49 +53,65 @@ matrix:
     - python: 3.5
       env: TOXENV=coverage
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest
     - python: 3.5
       env: FIREFOX=nightly
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest-nightly
     - python: 3.5
       env: FIREFOX=dev
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest-dev
     - python: 3.5
       env: FIREFOX=beta
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest-beta
     - python: 3.5
       env: FIREFOX=release
       addons:
+        apt:
+          packages:
+            - jwm
         firefox: latest
     - python: 3.5
       env: FIREFOX=esr
       addons:
+        apt:
+          packages:
+            - jwm
```

Comment:
I think the issue is that we want to override specific addons in the matrix. By specifying any addons in the matrix I believe we're replacing any defaults, so this is the only way to do what we need for now.